vmware-event-router/README.MD

```diff
@@ -298,7 +301,22 @@ as an event `processor`.
 | `region`   | String | AWS region to use, see [regions doc](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html). | true     | `us-west-1`                                                            |
 | `eventBus` | String | Name of the event bus to use                                                                                                            | true     | `default` or `arn:aws:events:us-west-1:1234567890:event-bus/customBus` |
 | `ruleARN`  | String | Rule ARN to use for event pattern matching                                                                                              | true     | `arn:aws:events:us-west-1:1234567890:rule/vmware-event-router`         |
-| `<auth>`   | Object | AWS IAM role credentials                                                                                                                | true     | (see `aws_access_key` example below)                                   |
+| `<auth>`   | Object | AWS IAM role credentials                                                                                                                | true     | (see `aws_access_key` example below)
+
+
+                                   |
+
+### processor-type-knative
+Knative is a Kubernetes-based platform to deploy and manage modern serverless workloads. Knative eventing supports late-binding event sources and event consumers, to enable delivery to multiple services. It also supports brokers and triggers to make it easy to filter events based on event attributes.
+
+The following table lists allowed and optional fields for using knative
+as an event `processor`.
+
+| Field      | Type   | Description                                                                                                                             | Required | Example                                                                |
+| ---------- | ------ | --------------------------------------------------------------------------------------------------------------------------------------- | -------- | ---------------------------------------------------------------------- |
+| `address`   | String | Knative Event Broker to use. | true     | `http://broker-ingress.knative-eventing.svc.cluster.local/default/default`                                                            |
```

Comment:
The one thing we might want to do here instead of hard coding to the broker is think about using Addressable as a general type. For example, if we do that, you could target something a Knative Serving Service directly, or a Channel, or ... :) 

We have couple of duck types that might make it easier to do this:
https://github.com/knative/pkg/blob/master/apis/duck/v1/destination.go#L26
https://github.com/knative/pkg/blob/master/resolver/addressable_resolver.go#L48

This way the user can specify the Resource they target or put in a fully specified URL or use a combination of the two.

For example (just picked one :) )
https://knative.dev/docs/eventing/samples/sequence/sequence-terminal/#create-the-pingsource-targeting-the-sequence

Let me know if that doesn't make sense :)